app/swagger/schemas/gibct/institution_programs.rb

```diff
@@ -11,9 +11,9 @@ class InstitutionPrograms
 
           property :data, type: :array, minItems: 0, uniqueItems: true do
             items do
-              property :id, type: :integer, example: '1234'
-              property :value, type: :string, example: '1X12345'
-              property :label, type: :string, example: 'COMPUTER SCIENCE'
+              property :id, type: :integer, example: '1234', description: 'Unique program ID'
```

Comment:
I'm looking at requests in my browser's developer tools.  Looks like these IDs actually return as integers.  i.e. `1234` instead of `'1234'`

Reply:
@ksantiagomorgan6 comment still stands.  This example should be `1234` (not `'1234'`) to reflect actual behavior.

Reply:
Updated